en/flight_controller/crazyflie2.md

```diff
@@ -17,134 +17,152 @@ The Crazyflie line of micro quads was created by Bitcraze AB. An overview of the
 
 ## Where to Buy
 
-* [Crazyflie 2.0](https://store.bitcraze.io/collections/kits/products/crazyflie-2-0)
-* [Crazyradio PA 2.4 GHz USB dongle](https://store.bitcraze.io/collections/kits/products/crazyradio-pa): used for wireless communication between *QGroundControl* (QGC) and CF2.
+* [Crazyflie 2.0](https://store.bitcraze.io/collections/kits/products/crazyflie-2-0).
+* [Crazyradio PA 2.4 GHz USB dongle](https://store.bitcraze.io/collections/kits/products/crazyradio-pa): used for wireless communication between *QGroundControl* and Crazyflie 2.0.
 * [Breakout deck](https://store.bitcraze.io/collections/decks/products/breakout-deck): breakout expansion board for connecting new peripherals. 
 * [Flow deck](https://store.bitcraze.io/collections/decks/products/flow-deck): contains an optical flow sensor to measure movements of the ground and a distance sensor to measure the distance to the ground. This will be useful for precise altitude and position control.
 * [Z-ranger deck](https://store.bitcraze.io/collections/decks/products/z-ranger-deck) has the same distance sensor as the Flow deck to measure the distance to the ground. This will be useful for precise altitude control.
 * [SD-card deck](https://store.bitcraze.io/collections/decks/products/sd-card-deck): used for high speed onboard logging to a micro SD card.
-* [Logitech Joystick](https://www.logitechg.com/en-ch/product/f310-gamepad)
+* [Logitech Joystick](https://www.logitechg.com/en-ch/product/f310-gamepad).
 
 ## Flashing PX4
 
-After setting up the PX4 development environment, follow these steps to put the PX4 software on the CF2:
+After setting up the PX4 development environment, follow these steps to install the PX4 Autopilot on the Crazyflie 2.0:
 
-1. Grab source code of the PX4 [Bootloader](https://github.com/PX4/Bootloader)
-1. Compile it using:
+1. Download the source code of the PX4 Bootloader:
+   ```
+   git clone https://github.com/PX4/Bootloader.git
+   ```
+1. Navigate into the top directory of the source code and compile it using:
    ```
    make crazyflie_bl
    ```
-1. Put the CF2 into DFU mode by following these steps:
+1. Put the Crazyflie 2.0 into DFU mode by following these steps:
    - Ensure it is initially unpowered.
-   - Hold down button.
+   - Hold down the reset button (see figure below...).
```

Comment:
Which figure? We can link to figure if it is not obvious.

Reply:
It would be nicer if we can have a close-up figure where we can see the reset button. 
I don't have the figure yet, but I am going to add one.

Reply:
That would be awesome!

Reply:
New image merged in https://github.com/PX4/px4_user_guide/pull/290